code/index.html

```diff
@@ -8,26 +8,31 @@
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet" />
-    <title>Chat Bot</title>
+    <title>Doggo Bot</title>
   </head>
 
   <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
+    <h1>Welcome to Doggo Bot.</h1>
+    <h2>What doggo is your doggo?</h2>
     <main>
-      <section class="chat" id="chat"></section>
-
-      <div class="input-wrapper" id="input-wrapper">
+      <section class="chat" id="chat"></section>    
+       <div class="input-wrapper" id="input-wrapper">
+         <button id="yes-btn">Hell yes!</button>
+         <button id="no-btn">Fy fan nej!</button>
```

Comment:
Remember that you will be showing these projects to the hiring managers and you never know how sensitive will them be. Try to keep your language clean 😸